src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/SearchObject.cs

```diff
@@ -286,5 +290,32 @@ private static Uri CreateEmbeddedLicenseUri(string packagePath, LicenseMetadata
 
             return fullUri;
         }
+
+        private IReadOnlyList<KnownOwner>? CreateKnownOwners(IPackageSearchMetadata packageSearchMetadata)
+        {
+            if (_ownerDetailsUriService is null
+                || !_ownerDetailsUriService.SupportsKnownOwners)
+            {
+                return null;
+            }
+
+            IReadOnlyList<string> ownersList = packageSearchMetadata.OwnersList;
+
+            if (ownersList is null || ownersList.Count == 0)
+            {
+                return Array.Empty<KnownOwner>();
+            }
+
+            List<KnownOwner> knownOwners = new(capacity: ownersList.Count);
+
+            foreach (string owner in ownersList)
+            {
+                Uri ownerDetailsUrl = _ownerDetailsUriService.GetOwnerDetailsUri(owner);
+                KnownOwner knownOwner = new(owner, ownerDetailsUrl);
+                knownOwners.Add(knownOwner);
```

Comment:
Variable only used once so we can remove it.

```suggestion
                knownOwners.Add(new KnownOwner(owner, ownerDetailsUrl));
```